getting-started/release-notes/changes-in-timescaledb-2.md

```diff
@@ -0,0 +1,290 @@
+
+**This differs significantly from how TimescaleDB 1.x handled conflicts between retention policies and continuous aggregates.**
+
+Previously, if the continuous aggregation setting `ignore_invalidation_older_than` overlapped with data that would be dropped by a retention policy, the retention policy would silently fail.  Making the retention policy work again required users to modify settings in either the retention policy or the continuous aggregate, and even then some data wasn’t always materialized as expected. 
+
+After upgrading to TimescaleDB 2.0, **retention policies will no longer fail due to incompatibilities with continuous aggregates** and users have to ensure that retention and continuous aggregate policies have the desired interplay. 
+
+## Differences in the handling of backfill on continuous aggregates
+
+In TimescaleDB 1.x, data that was backfilled into hypertables wasn’t handled optimally; modification of any hypertable data, regardless of how old, would cause the continuous aggregate materializer job to restart from the point of the earliest backfill and then it would materialize from that point forward. Unfortunately, this could also cause the materializer to get “stuck”, since the background job only processed a limited amount of data per run, as governed by the `max_interval_per_job` setting. When this happened, one job run could potentially force the next job to start all over again. 
+
+To prevent this, the `ignore_invalidation_older_than` setting could be used to ignore backfill data older than a specified interval (e.g., ignore backfill older than 1 week), preventing the materializer from restarting if hypertable data was modified beyond this interval boundary. However, this would also stop TimescalDB 1.x from tracking changes in the underlying data beyond the `ignore_invalidation_older_than` threshold too. This meant that it was not possible to revert this setting later to a larger interval (e.g., 1 month) without potentially having a mismatch between the raw data and the aggregated data in the continuous aggregate.
+
+In contrast, TimescaleDB 2.0 never stops tracking backfill, and to avoid materializing too much historical data, one should simply use a refresh window that does not include that region of data. The backfilled region can always be refreshed at a later time, either manually or via a policy.
+
+To ensure that previously ignored backfill can be refreshed after the upgrade to TimescaleDB 2.0, the upgrade process will mark the region older than the `ignore_invalidation_older_than` threshold as “requiring refresh”. This allows a manual refresh to bring a continuous aggregate up-to-date with the underlying source data. If the `ignore_invalidation_older_than` threshold was reset to a larger interval at some point, we recommend setting it back to the smaller interval prior to upgrading to ensure that all the backfill can be refreshed, if one so desires. 
```

Comment:
Not sure what 'larger interval' is referring to here?

Reply:
See what you think of this minor change.